t/lib/feature/implicit

```diff
@@ -58,41 +58,3 @@ Helloworld
 # no implicit features with 'no'
 eval "no " . ($]+1); print $@;
 EXPECT
-########
-# lower version after higher version
-sub evalbytes { print "evalbytes sub\n" }
-sub say { print "say sub\n" }
-use 5.015;
-evalbytes "say 'yes'";
-use 5.014;
-evalbytes;
-use 5;
-say "no"
-EXPECT
-Downgrading a use VERSION declaration to below v5.11 is deprecated, and will become fatal in Perl 5.40 at - line 8.
-yes
-evalbytes sub
-say sub
-########
-# Implicit unicode_string feature
```

Comment:
I think all three of these tests are supposed to be testing a change before/after v5.15 and the addition of the downgrade warning was added to the tests when we added the warning. But removing this warning and making it fatal isn't a reason to get rid of these tests - they are here to test other behaviour. I think the real solution is to reorder the tests like so:
```
# lower version after higher version
sub evalbytes { print "evalbytes sub\n" }
sub say { print "say sub\n" }
use 5;
say "no";
use 5.015;
evalbytes "say 'yes'";
use 5.014;
evalbytes;
EXPECT
say sub
yes
evalbytes sub
########
# Implicit unicode_string feature
use v5.8.8;
my $sharp_s = chr utf8::unicode_to_native(0xdf);
print 'ss' =~ /$sharp_s/i ? "ok\n" : "nok\n";
use v5.14;
print 'ss' =~ /$sharp_s/i ? "ok\n" : "nok\n";
EXPECT
nok
ok
########
# Implicit unicode_eval feature
require '../../t/charset_tools.pl';
my $long_s = byte_utf8a_to_utf8n("\xc5\xbf");
use v5.8.8;
print eval "use utf8; q|$long_s|" eq "\x{17f}" ? "ok\n" : "nok\n";
use v5.15;
print eval "use utf8; q|$long_s|" eq $long_s ? "ok\n" : "nok\n";
EXPECT
ok
ok

```

Reply:
Ahyes; that looks a much better idea. I've done that instead in 517020f627